optuna/distributions.py

```diff
@@ -251,7 +251,7 @@ class DiscreteUniformDistribution(FloatDistribution):
         :math:`\\mathsf{high}` will be replaced with the maximum of :math:`k q + \\mathsf{low}
         < \\mathsf{high}`, where :math:`k` is an integer.
 
-    Attributes:
+    Args:
```

Comment:
`q` in `Attributes` conflicts with the docstring of property `q` showing the message:

```
/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/optuna/distributions.py:docstring of optuna.distributions.DiscreteUniformDistribution.q:1: WARNING: duplicate object description of optuna.distributions.DiscreteUniformDistribution.q, other instance in reference/generated/optuna.distributions.DiscreteUniformDistribution, use :noindex: for one of them
```

And, docstrings for distributions lacks `Args` section. This PR only changes `DiscreteUniformDistribution` but it may be better to update other docstrings as well.